conjure-go-contract/errors/unmarshal.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	"reflect"
+
+	"github.com/palantir/conjure-go-runtime/conjure-go-contract/codecs"
+	werror "github.com/palantir/witchcraft-go-error"
+)
+
+// UnmarshalError attempts to deserialize the message to a known implementation of Error.
+// Custom error types should be registered using RegisterErrorType.
+// If the ErrorName is not recognized, a genericError is returned with all unsafe params.
```

Comment:
"is returned with all params marked unsafe"?

Reply:
done